api/v1alpha1/bundle_types.go

```diff
@@ -80,6 +80,11 @@ type GitSource struct {
 	// is required. Setting more than one field or zero fields will result in an
 	// error.
 	Ref GitRef `json:"ref"`
+	// Secret contains the secret name that has authorization information for HTTPS protocol and is in the namespace that the provisioner is deployed.
+	// The secret contains the user ID (key: user) and the personal access token to the repository (key: token)
+	Secret string `json:"secret,omitempty"`
```

Comment:
Rename to `SecretName`?

I noticed that the `pod.spec.serviceAccount` field is deprecated in favor of `pod.spec.serviceAccountName`. Seems like the reasoning for that applies here as well.

Reply:
Yes, I'll update.  Thanks!